Products/ZenModel/migrate/shardGuidTable.py

```diff
@@ -0,0 +1,34 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2017, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+import Migrate
+
+from Products.ZenUtils.ShardedBTree import ShardedBTree
+from Products.ZenUtils.guid.guid import GUID_TABLE_PATH
+
+
+class ShardGuidTable(Migrate.Step):
+    version = Migrate.Version(112, 0, 0)
+
+    def cutover(self, dmd):
+        old_table = dmd.unrestrictedTraverse(GUID_TABLE_PATH)
+        if not isinstance(old_table, ShardedBTree):
+            # get table parent and name
+            splitted_path = GUID_TABLE_PATH.split("/")
+            table_parent = dmd.unrestrictedTraverse(splitted_path[:-1])
+            table_name = splitted_path[-1]
+            # create sharded table
+            sharded_tree = ShardedBTree()
+            sharded_tree.update(old_table)
+            # replace the old table
+            table_parent._delOb(table_name)
```

Comment:
Please add a comment as to why `_delOb` is being used as opposed to `_delObject`.